stdlib/printexc.ml

```diff
@@ -61,14 +74,13 @@ let to_string_default = function
       sprintf locfmt file line char (char+6) "Assertion failed"
   | Undefined_recursive_module(file, line, char) ->
       sprintf locfmt file line char (char+6) "Undefined recursive module"
+  | CamlinternalEffect.Unhandled e ->
+      sprintf "Stdlib.Effect.Unhandled(%s)"
+        (string_of_sum_type (walk_sum_type (Obj.repr e)))
   | x ->
```

Comment:
Maybe there should also be a special case for `Effect.Continuation_already_taken`. More precisely,
```ocaml
  | Effect.Continuation_already_taken -> "Continuation already taken"
```
Actually, should it be `Continuation_already_resumed` for perfect consistency with the API documentation? Also https://github.com/ocaml/ocaml/issues/11422 might be related.

Reply:
Indeed. `Continuation_already_resumed` feels more appropriate than "taken".